Graylog2/graylog2-server
# Disable users after deactivating auth backend

/jenkins-pr-deps Graylog2/graylog-plugin-enterprise#5562

## Description
Makes the `ActiveAuthServiceBackendChangedEvent` aware of previously activated backend to disable users.

## Motivation and Context
fixes #13823 

## How Has This Been Tested?
- Add and activate new LDAP authentication backend
- Login as LDAP user
- Deactivate backend
- User should be disabled
- Add another backend (e.g. Okta)
- Login as user from this backend
- Switch backends by re-enabling LDAP
- Okta user should be disabled
- Login as LDAP user
- LDAP user should be reenabled

## Types of changes


## Checklist:
